### PR TITLE
Refresh Code base

### LINKED ISSUE #15: Any plan to migrate this package to Laminas ?
hello, @weierophinney.  I would appreciate it if you can migrate this package as this package provide me some use case for static site generator kind of concept and this helps to keep`Laminas` is my first priority for development.

### PR DESCRIPTION
**Pull request by rieschl**
* remove custom autoloaders
* migrate to Laminas
* fix test namespace
* (mostly) apply Laminas Coding Standard
* add GitHub Action
* upgrade to laminas-mvc 3 and laminas-eventmanager 3

Sorry this is a very very big PR. Normally I don't do that because reviewing is quite hard that way, but the code base is very old and probably has a low usage, it's much easier than separate PRs.
But the commits are somewhat divided: the first one is the big one with Laminas, code style and code quality changes and the upgrade from mvc and eventmanager v2 to v3 is in a separate commit. The commits after that are bug fixes and forgotten stuff.

Edit: ... and I tested it on a freshly installed laminas-mvc skeleton 🙂 

fixes #15

### CODE REVIEW

**Comment by rieschl** [comment]
Hmm, are GitHub Actions disabled in this repository? The pull_request event should trigger the new action here. The push action worked in my fork https://github.com/rieschl/PhlyBlog/actions

File: composer.json [hunk 1]
```diff
@@ -17,16 +18,51 @@
...
+        "laminas/laminas-inputfilter": "^2.5",
+        "laminas/laminas-modulemanager": "^2.5",
+        "laminas/laminas-mvc": "^3.0",
+        "laminas/laminas-paginator": "^2.5",
+        "laminas/laminas-stdlib": "^3.0",
+        "laminas/laminas-tag": "^2.5",
+        "laminas/laminas-uri": "^2.5",
+        "laminas/laminas-validator": "^2.5",
+        "laminas/laminas-view": "^2.5",
+        "phly/phly-common": "dev-refresh"
```
Comment (rieschl) [inline composer.json]: This line and the `repositories` block will be removed after `PhlyCommon` was merged and tagged.

File: README.md [hunk 2]
```diff
@@ -31,9 +31,8 @@ project:
 
 ```js
 {
-    "minimum-stability": "dev",
     "require": {
-        "phly/phly-blog": "dev-master"
+        "phly/phly-blog": "^3.0"
```
Comment (rieschl) [inline README.md:35]: It'll be tagged as `3.0.0`, right?

**Comment by vrkansagara** [comment]
@rieschl  and @weierophinney this looks good to me. :100:

File: README.md (hunk 2 above)
Comment (weierophinney) [reply README.md:35]: I never did a 2.0, so this would be 2.0 here. :)

**Comment by weierophinney** [comment]
@rieschl Actions have to be present in the default branch in order to run, IIRC.

**Pull request merged**